pkg/utils/utils.go

```diff
@@ -344,7 +348,16 @@ func GetS3PETMFromParamsMap(params map[string]interface{}, logger logrus.FieldLo
 	// If the credentials are explicitly provided in params, use it.
 	// else let aws API pick the default credential provider.
 	var sess *session.Session
-	if _, ok := params[constants.AWS_ACCESS_KEY_ID]; ok {
+	var sessionOptions session.Options
+	caCert, ok := GetStringFromParamsMap(params, "caCert", logger)
```

Comment:
Why do you place the code snippet about getting "caCert" here while it is not used until L373? Can we move it there?